appdatabase/migrations/sql/1674570455_communities_permissions.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS communities_permissions (
+    community_id TEXT NOT NULL,
+    permission_id VARCHAR NOT NULL PRIMARY KEY,
+    hide BOOLEAN NOT NULL DEFAULT FALSE,
+    is_allowed_to INT NOT NULL DEFAULT 0,
+    holds_tokens BOOLEAN NOT NULL DEFAULT FALSE 
+);
+
+CREATE TABLE IF NOT EXISTS communities_permissions_chats {
```

Comment:
I wonder if the chat's permissions can be merged in the communities table. It would simplify queries and speed it up too. Especially since both the whole community and simple channels can have permissions. Maybe just need a `type` to differentiate the two

Reply:
We have 2 condition:
1. If `is_allowed_to` is `become_member` or `become_admin` we should ignore chats
2. if `is_allowed_to` is another value - we must have to get chat_ids from this table.

I add a new table for it, cos storing permission_ids in `chats` table as json isn't comfortable for using.